cloudsim-plus/src/main/java/org/cloudsimplus/heuristics/CloudletToVmMappingSolution.java

```diff
@@ -345,11 +345,10 @@ boolean swapVmsOfTwoRandomSelectedMapEntries() {
         until finding the required entries, without creating
         a List with all entries in order to get just two of them.
```

Comment:
Please fix some issues in the comment, replacing it by:

Loop over the entries until the entries defined by the first and second index
are found and added to the List.
Since Map doesn't have an index, we can't access the ith entry directly.
This loop ensures we iterate the least number of times
until finding the required entries, without creating
a List with all entries in order to get just two of them.

Reply:
Done.